### PR TITLE
Option[Seq[String]] fix

### PR DESCRIPTION
**Pull request by natefrechette**
When parsing a type as Option[Seq[String]], the item type resulted in "String]", I believe because the itemType0 method was picking up the Seq[String] inside the option.  Changing the order in the else if fixed the issue and now yields item type "String"

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -3,6 +3,8 @@ organization in ThisBuild := "com.iheart"
 
 name := "play-swagger"
 
+version := "0.1.7"
```
Comment (kailuowang) [inline build.sbt]: This shouldn't be here. we use sbt-release plugin to manage version, which stores version in version.sbt

File: src/main/scala/com/iheart/playSwagger/SwaggerParameterMapper.scala [hunk 2]
```diff
@@ -50,14 +50,17 @@ object SwaggerParameterMapper {
         case "Long"                  ⇒ swaggerParam("integer", Some("int64"))
         case "Double"                ⇒ swaggerParam("number", Some("double"))
         case "Float"                 ⇒ swaggerParam("number", Some("float"))
+        case "String"                ⇒ swaggerParam("string")
```
Comment (kailuowang) [inline src/main/scala/com/iheart/playSwagger/SwaggerParameterMapper.scala]: This is probably not needed as the `unknown ` case does exactly the same thing.

**Comment by kailuowang** [comment]
@natefrechette Thanks very much for your contribution. I think it will be nice to have a test exposing the bug you originally found. How about adding the following test to `SwaggerParameterMapperSpec`?
```

    "map Option[Seq[T]] to item type" >> {
      mapParam("aField", "Option[Seq[String]]") === SwaggerParameter(
        name = "aField",
        required = false,
        items = Some("String")
      )
    }
```
This test fails without your swap change but will pass with it. 
Thanks again!

File: src/main/scala/com/iheart/playSwagger/SwaggerParameterMapper.scala (hunk 2 above)
Comment (natefrechette) [inline src/main/scala/com/iheart/playSwagger/SwaggerParameterMapper.scala]: Agreed I will add the test and strong is not necessary.  I added it when
trying to figure out what was causing the option bug.  Will submit again
tomorrow!  Thanks

Nate

On Wednesday, November 25, 2015, Kailuo Wang <notifications@github.com>
wrote:

> In src/main/scala/com/iheart/playSwagger/SwaggerParameterMapper.scala
> <https://github.com/iheartradio/play-swagger/pull/29#discussion_r45936693>
> :
>
> > @@ -50,14 +50,17 @@ object SwaggerParameterMapper {
> >          case "Long"                  ⇒ swaggerParam("integer", Some("int64"))
> >          case "Double"                ⇒ swaggerParam("number", Some("double"))
> >          case "Float"                 ⇒ swaggerParam("number", Some("float"))
> > +        case "String"                ⇒ swaggerParam("string")
>
> This is probably not needed as the unknown case does exactly the same
> thing.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/iheartradio/play-swagger/pull/29/files#r45936693>.
>


-- 
*Nate Frechette*
Chief Technology Officer & Co-founder

(315) 420-4252
nate@dropsource.com
dropsource.com

**Comment by kailuowang** [comment]
@natefrechette thanks again for your contribution, I just made a new release.
https://bintray.com/iheartradio/maven/play-swagger/0.1.9/view

**Comment by natefrechette** [comment]
Happy to help!

**Pull request merged**